lib/baz/collection_event_imports/jobs/pull_collection_events_by_page.ex

```diff
@@ -83,17 +83,16 @@ defmodule Baz.CollectionEventImports.Jobs.PullCollectionEventsByPage do
           page_number: input.page_number,
           next_page_cursor: page.next_page_cursor
         }
-
         import_page = Ecto.build_assoc(input.import, :pages, import_page_attrs)
-        multi = Ecto.Multi.insert(multi, :import_page, import_page)
+        multi = Ecto.Multi.insert(multi, :import_page, import_page, on_conflict: :nothing)
```

Comment:
Do you have an example case where the `import_page` causes a conflict?

Reply:
Yeah, I captured this log.  I saw a lot of them at some point:
```22:25:56.462 [info] pull collection events by page=85, venue=open_sea, slug=azuki, token_ids=nil, import_id=6
22:25:56.489 [error] unhandled error pulling collection events by page venue=open_sea, slug=azuki, token_ids=nil, import_id=6, error=%Ecto.ConstraintError{constraint: "collection_event_import_pages_collection_event_import_id_page_n", message: "constraint error when attempting to insert struct:\n\n    * collection_event_import_pages_collection_event_import_id_page_n (unique_constraint)\n\nIf you would like to stop this constraint violation from raising an\nexception and instead add it as an error to your changeset, please\ncall `unique_constraint/3` on your changeset with the constraint\n`:name` as an option.\n\nThe changeset has not defined any constraint.\n", type: :unique}, stacktrace=[{Ecto.Repo.Schema, :"-constraints_to_errors/3-fun-1-", 4, [file: 'lib/ecto/repo/schema.ex', line: 783]}, {Enum, :"-map/2-lists^map/1-0-", 2, [file: 'lib/enum.ex', line: 1593]}, {Ecto.Repo.Schema, :constraints_to_errors, 3, [file: 'lib/ecto/repo/schema.ex', line: 768]}, {Ecto.Repo.Schema, :apply, 4, [file: 'lib/ecto/repo/schema.ex', line: 749]}, {Ecto.Repo.Schema, :"-do_insert/4-fun-1-", 15, [file: 'lib/ecto/repo/schema.ex', line: 367]}, {Ecto.Multi, :apply_operation, 5, [file: 'lib/ecto/multi.ex', line: 794]}, {Enum, :"-reduce/3-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 2396]}, {Ecto.Multi, :"-apply_operations/5-fun-1-", 5, [file: 'lib/ecto/multi.ex', line: 768]}]
22:25:56.506 [info] pull collection events by page=85, venue=open_sea, slug=azuki, token_ids=nil, import_id=6
22:25:56.513 [error] unhandled error pulling collection events by page venue=open_sea, slug=azuki, token_ids=nil, import_id=6, error=%Ecto.ConstraintError{constraint: "collection_event_import_pages_collection_event_import_id_page_n", message: "constraint error when attempting to insert struct:\n\n    * collection_event_import_pages_collection_event_import_id_page_n (unique_constraint)\n\nIf you would like to stop this constraint violation from raising an\nexception and instead add it as an error to your changeset, please\ncall `unique_constraint/3` on your changeset with the constraint\n`:name` as an option.\n\nThe changeset has not defined any constraint.\n", type: :unique}, stacktrace=[{Ecto.Repo.Schema, :"-constraints_to_errors/3-fun-1-", 4, [file: 'lib/ecto/repo/schema.ex', line: 783]}, {Enum, :"-map/2-lists^map/1-0-", 2, [file: 'lib/enum.ex', line: 1593]}, {Ecto.Repo.Schema, :constraints_to_errors, 3, [file: 'lib/ecto/repo/schema.ex', line: 768]}, {Ecto.Repo.Schema, :apply, 4, [file: 'lib/ecto/repo/schema.ex', line: 749]}, {Ecto.Repo.Schema, :"-do_insert/4-fun-1-", 15, [file: 'lib/ecto/repo/schema.ex', line: 367]}, {Ecto.Multi, :apply_operation, 5, [file: 'lib/ecto/multi.ex', line: 794]}, {Enum, :"-reduce/3-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 2396]}, {Ecto.Multi, :"-apply_operations/5-fun-1-", 5, [file: 'lib/ecto/multi.ex', line: 768]}]
22:25:56.528 [info] pull collection events by page=84, venue=open_sea, slug=azuki, token_ids=nil, import_id=6
```

Reply:
This seems like a different bug. Possibly related to retries. Can you :scissors: from this PR and I will investigate in a bug ticket

Reply:
Sure, but considering there is related index, this is a must.  Even if there is a different unrelated bug.

Reply:
Removed from this PR